css/search.css

```diff
@@ -1,7 +1,9 @@
-li.jobs-unified-top-card__job-insight--highlight,
+.jobs-unified-top-card__job-insight--highlight,
```

Comment:
I fiddled around with this because it hides the verification section but couldn't find a solution - the verification section and then "compare yourself to other applicants" section both have the same structure in the html and the same classes.